components/markdown_image.jsx

```diff
@@ -113,9 +113,7 @@ export default class MarkdownImage extends React.PureComponent {
                                 rel='noopener noreferrer'
                                 target='_blank'
                                 title={this.props.title}
-                            >
-                                {alt}
-                            </a>
+                            />
```

Comment:
I don't know if this needs to be changed. If someone links to an unsafe image, it won't show anything now whereas it made a link to that image before.

Reply:
Oh I see, thanks for clarifying